lib/Header/Header.js

```diff
@@ -796,9 +797,31 @@ module.exports = kind(
 	* @fires module:moonstone/Header~Header#onInputHeaderChange
 	* @private
 	*/
-	handleChange: function (inSender, inEvent) {
-		this.doInputHeaderChange({originalEvent: util.clone(inEvent, true)});
-	},
+    handleChange: function (inSender, inEvent) {
+        this.doInputHeaderChange({originalEvent: util.clone(inEvent, true)});
+        this.$.titleInputMarquee.setContent(event.originator.getValue());
```

Comment:
This should also reference the event parameter, which is `inEvent` in this case.